javascript/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@jupyter-notebook/ydoc",
```

Comment:
Would it make sense to use the `@jupyter-server` scope instead?

Reply:
This package will most likely end up in Jupyter GitHub organization as it is a formal description of documents (similar to nbformat but applied on CRDT). This is the reason I went for `jupyter-notebook` organization (as we don't own `jupyter` on npm) rather than `jupyter-server`.

Reply:
Thanks for the invitation